tests/data/validate/optional-base-uri-params.raml

```diff
@@ -24,11 +24,7 @@ uriParameters:
       example: "foo-api"
   get:
     description: "Get some foo"
-    type: "string"
```

Comment:
This was throwing a `MultipleValidationErrors` exception because you're defining a type on an HTTP method.  It was unrelated to the test, in my opinion.